src/content/app/genome-browser/hooks/useGenomeBrowser.ts

```diff
@@ -151,6 +152,78 @@ const useGenomeBrowser = () => {
     }
   };
 
+  const restoreTrackConfigStates = () => {
```

Comment:
Doesn't this function, with all the complexities inside it, belong in the `useBrowserTrackConfig` hook?

Reply:
✅

Reply:
Decided against it.